src/NavBar.module.css

```diff
@@ -0,0 +1,67 @@
+.nav {
+  height: 100%;
+  background: var(--colors-bgPrimary);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: var(--spacing-md);
```

Comment:
Increased from spacing-sm to allocate space for the tooltip 

![Screenshot 2025-03-18 at 9 24 34 AM](https://github.com/user-attachments/assets/2c47a4d7-3215-44a2-9d17-b93eaa7bc49d)